src/OpenTok/Exception/SignalDomainException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace OpenTok\Exception;
+
+// TODO: this kind of exception has a detailed message from the HTTP response, use it.
+/**
+* Defines an exception thrown when a call to a signal method results in an error response from
+* the server.
+*/
+class SignalDomainException extends \OpenTok\Exception\DomainException implements \OpenTok\Exception\SignalException
+{
+}
```

Comment:
I think we should add exception classes for the following:

* SignalConnectionException -- The client specified by the connectionId is not connected to the session.
* SignalLengthException (or some better name) -- The type string exceeds the maximum length (128 bytes), or the data string exceeds the maximum size (8 kB). (Or should SignalUnexpectedValueException cover this?)

Reply:
@jeffswartz I like the idea of `SignalConnectionException`, but for the second one, I think we can have `SignalUnexpectedValueException` which would catch the properties and maximum length